mlflow/sagemaker/__init__.py

```diff
@@ -15,6 +15,17 @@
 
 DEFAULT_IMAGE_NAME = "mlflow_sage"
 
+PREBUILT_IMAGE_URLS = {
+    "us-west-2": "XXXXXXX.dkr.ecr.us-west-2.amazonaws.com/mlflow-pyfunc",
+}
+
+def _get_prebuilt_image_url(region):
+    if region not in PREBUILT_IMAGE_URLS:
+        return None
+    return (PREBUILT_IMAGE_URLS[region] + ":{version}").format(version=mlflow.version.NEXT_VERSION)
```

Comment:
it's awkward here whether the current or next version should be used.
* why use next version:
  * the current version may be outdated if any changes have happened to the container code in the repository.
  * if we add new images (aside from pyfunc), there would not exist a current version for the image (though we could add it manually)
* why use current version:
  * there is no guarantee the image for the next version exists.
  * it's unlikely there will be a lot of changes made to the container code
  * we may not actually know what the next version number is
Perhaps it's simplest to stick with the current version and be vigilant about integration tests.

Reply:
Yeah those are good questions. My opinion is it would be easiest to always use mlflow.version.VERSION. That way we do not have to be updating version on multiple places e.g. when doing release. We can still get either of the above behaviors by setting version in MLflow to the next one (or having special dev version) and then either publishing public image or not.